### PR TITLE
Appgwv2 changes

### PR DESCRIPTION
**Pull request by akshaysngupta**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by akshaysngupta** [comment]
Can we merge this PR for mid-June release ?

File: src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs [hunk 1]
```diff
@@ -149,6 +149,16 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
             HelpMessage = " Whether HTTP2 is enabled.")]
         public SwitchParameter EnableHttp2 { get; set; }
 
+        [Parameter(
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs]: Do we need to update the related tests to use the new params here?

**Comment by praries880** [comment]
need to wait for [this change](https://github.com/Azure/azure-rest-api-specs/commit/420f89e28854bfae7f1d324a173bdf65515d973d) to make it out to a newer Network SDK before we can merge this PR

File: src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs [hunk 2]
```diff
@@ -149,6 +149,16 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
...
+            Mandatory = false,
+            HelpMessage = " Whether FIPS is enabled.")]
+        public SwitchParameter EnableFIPS { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A list of availability zones denoting where the application gateway needs to come from.")]
+        public List<string> Zones { get; set; }
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs]: This param name needs to be singular, _Zone_ (even if the user can specify multiple zones, theres always the case where they'll specify just one)

Comment (akshaysngupta) [reply src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs]: https://github.com/Azure/azure-powershell/blob/796f80815e11eaa46ad728c09e2beda88e1de892/src/ResourceManager/Network/Commands.Network/Models/PSPublicIpAddress.cs#L38

Comment (praries880) [reply src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs]: @akshaysngupta This is an enforced [PS guideline](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#noun-singularity). The plural name is fine in the PSObject, the corresponding param however needs to be singular.

**Comment by maddieclayton** [comment]
@akshaysngupta Ping on the comment above.

**Comment by maddieclayton** [comment]
@akshaysngupta Can you please fix the merge conflict? We have moved ChangeLog/psd1 files from src/ResourceManager/{Module} to src/ResourceManager/{Module}/Commands.{Module}, so you will need to move your changelog/psd1 updates there.

**Comment by akshaysngupta** [comment]
I waiting for the merge in azure rest-api-spec

**Comment by cormacpayne** [comment]
Closing this PR in favor of https://github.com/Azure/azure-powershell/pull/6525

**Comment by akshaysngupta** [comment]
Thanks Cormac!

**Pull request closed without merging**